packages/cactus-plugin-satp-hermes/src/main/typescript/core/stage-services/data-verifier.ts

```diff
@@ -0,0 +1,200 @@
+import { JsObjectSigner } from "@hyperledger/cactus-common";
```

Comment:
can we have several verifier methods on the SATP session class? Since these verificartions seem tighly coupled to the session, perhaps it makes more sense to add this functionality rather than having it as functions